From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Athenas Jimenez <athenas.jimenez.gonzalez@intel.com>
Date: Wed, 20 May 2015 17:18:14 +0000
Subject: [PATCH] Fix os_test in stateless

Change the dir /etc -> /usr/share/defaults/etc
The file "hosts" remplaced by "shadow"

Signed-off-by: Athenas Jimenez <athenas.jimenez.gonzalez@intel.com>
---
 src/os/os_test.go | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/os/os_test.go b/src/os/os_test.go
index 31b3e67..2a9325c 100644
--- a/src/os/os_test.go
+++ b/src/os/os_test.go
@@ -99,10 +99,10 @@ var sysdir = func() *sysDir {
 		}
 	}
 	return &sysDir{
-		"/etc",
+		"/usr/share/defaults/etc",
 		[]string{
 			"group",
-			"hosts",
+			"shadow",
 			"passwd",
 		},
 	}
